kernel/configs/powerpc_kata_kvm_4.14.x

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/powerpc 4.14.49 Kernel Configuration
+# Linux/powerpc 4.14.50 Kernel Configuration
```

Comment:
Nit: this seems to be not strictly part of this PR?

Reply:
@jodh-intel : For the CI to pass, do we not need to bump kernel version here and increment the kata-config-version?

Reply:
@jodh-intel : Also I think we should get jenkins run on ppc64le as well, so that the earlier patch merge Fixes: #111  would have failed on Power and got reported.

Reply:
You need to bump the value in `kernel/kata_config_version` but your commit says, "just rename". But this change implies you've also bumped the kernel version, which is a totally separate issue.

Reply:
oh - I think I see what you mean. You *should* be able to simply modify `kernel/kata_config_version` so please can you revert this change.